frontend/src/SearchView.svelte

```diff
@@ -0,0 +1,243 @@
+<style>
+  button {
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    padding-left: 1em;
+    padding-right: 1em;
+  }
+
+  button:hover,
+  button:focus {
+    outline: none;
+    box-shadow: inset 1px 1px 0 var(--main-fg-color),
+      inset -1px -1px 0 var(--main-fg-color);
+  }
+
+  button:active {
+    box-shadow: inset 2px 2px 0 var(--main-fg-color),
+      inset -2px -2px 0 var(--main-fg-color);
+  }
+
+  .container {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: stretch;
+    align-content: center;
+  }
+
+  .inputs {
+    flex-grow: 1;
+  }
+
+  .inputs *:first-child {
+    margin-top: 0;
+  }
+
+  .actions {
+    margin-top: 2em;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-evenly;
+    align-items: flex-start;
+    align-content: flex-start;
+  }
+
+  input {
+    background: inherit;
+    color: inherit;
+    border: none;
+    border-bottom: 1px solid white;
+    flex-grow: 1;
+    margin-left: 1ch;
+  }
+
+  input:focus {
+    outline: none;
+    box-shadow: inset 0 -1px 0 var(--main-fg-color);
+  }
+
+  /* Adapted from: https://moderncss.dev/pure-css-custom-checkbox-style/ */
+  input[type="checkbox"] {
+    flex-grow: 0;
+    margin-left: 1ch;
+    /*
+    appearance: none;
+    background-color: var(--main-bg-color);
+    margin: 0;
+    color: currentColor;
+    width: 1em;
+    height: 1em;
+    border: 1px solid currentColor;
+    display: grid;
+    place-content: center;
+    */
+  }
+
+  /*
+  input[type="checkbox"]:focus {
+    box-shadow: none;
+  }
+
+  input[type="checkbox"]::before {
+    content: "";
+    width: 0.45em;
+    height: 0.45em;
+    transform: scale(0);
+    transition: 120ms transform ease-in-out;
+    box-shadow: inset 1em 1em var(--main-fg-color);
+  }
+
+  input[type="checkbox"]:checked::before {
+    transform: scale(1);
+  }
+  */
```

Comment:
This commented-out CSS can be removed. 

Aside: on the OFRAK site, I made consistently-styled, cross-platform, cross-browser, accessible checkboxes. I've yet to port them to the GUI, but this would be a good thing to make a new issue for. Specifically, we'll want a "checkbox" Svelte component. 

For reference, see [the `ofrak.com` stylesheet](https://ofrak.com/style.css):

``` css
/* Taken from https://ofrak.com/style.css */

input[type="checkbox"] {
  margin: 0 1ch;
}

label.checkbox {
  flex-direction: row;
  align-items: center;
  max-width: min-content;
}

label.checkbox input[type="checkbox"] {
  display: none;
}

label.checkbox input[type="checkbox"] + span {
  width: 1em;
  height: 1em;
  background: var(--main-bg-color);
  color: var(--main-fg-color);
  border: 1px solid var(--main-fg-color);
  margin-left: 2ch;
  box-sizing: content-box;
}

label.checkbox input[type="checkbox"]:checked + span::after {
  content: "";
  display: block;
  position: relative;
  top: 0;
  bottom: 0;
  margin: 0.25em;
  width: 0.5em;
  height: 0.5em;
  color: var(--main-fg-color);
  background: var(--main-fg-color);
  border: none;
}
```

This CSS is associated with HTML like the following:

``` html
<label class="checkbox">
  Sign Up for Mailing List
  <input type="checkbox" />
  <span></span>
</label>

```

The checkbox svelte component will pretty much only include this HTML and CSS and a slot for the label, as well as a reactive variable with the state of whether it's checked or not.